### PR TITLE
Feature/ranking view

### PR DESCRIPTION
**Pull request by test0n3**
Styles and dummy content for ranking view

### CODE REVIEW

File: simcase-game/src/views/Ranking.js [hunk 1]
```diff
@@ -2,27 +2,190 @@
...
+  // width: "50%"
+};
+
+const col_3Style = {
+  width: "30%"
+};
+
+const imgTableStyle = {
+  width: 40
+  // marginRight: 10,
```
Comment (CesarCachay) [inline simcase-game/src/views/Ranking.js]: What do you think about deleting this commented line?

File: simcase-game/src/views/Ranking.js [hunk 2]
```diff
@@ -2,27 +2,190 @@
...
+              {players &&
+                Object.values(players).map((user, index) => {
+                  if (index < 5) {
+                    return (
+                      <tr key={user.username}>
+                        <td css={col_1Style}>{user.id}</td>
+                        <td
+                          css={{
+                            ...col_2Style,
+                            textAlign: "left",
```
Comment (CesarCachay) [inline simcase-game/src/views/Ranking.js]: I think you could start working with this styles like you have in the top of the file creating consts

File: simcase-game/src/views/Ranking.js [hunk 3]
```diff
@@ -2,27 +2,182 @@
...
+                    backgroundColor: "#0C4785",
+                    color: "#FFFFFF",
+                    padding: 10
+                  }}
+                >
+                  #
+                </th>
+                <th
+                  scope="col"
+                  css={{
```
Comment (paulotijero) [inline simcase-game/src/views/Ranking.js]: Move all styles into variable const outside the function, and reuse styles

**Review by CesarCachay — CHANGES_REQUESTED** [review]

File: simcase-game/src/views/Ranking.js [hunk 4]
```diff
@@ -2,27 +2,182 @@
...
+  1: { id: 1, username: "ccachay", points: 961 },
+  2: { id: 2, username: "dconde", points: 871 },
+  3: { id: 3, username: "jwayne", points: 841 },
+  4: { id: 4, username: "wsmith", points: 771 },
+  5: { id: 5, username: "sstallone", points: 762 },
+  6: { id: 6, username: "dcuevas", points: 714 },
+  7: { id: 7, username: "jperez", points: 601 }
+};
 
+const contentStyle = {
```
Comment (paulotijero) [inline simcase-game/src/views/Ranking.js]: It's not necessary to add `style` to name const

File: simcase-game/src/views/Ranking.js [hunk 5]
```diff
@@ -2,27 +2,182 @@
...
+  fontSize: 18,
+  padding: 5
+};
+
+const col_1Style = {
+  width: "20%"
+};
+
+const col_2Style = {
+  // width: "50%"
```
Comment (paulotijero) [inline simcase-game/src/views/Ranking.js]: Be careful, don't forget to delete the commented line, and Isn't necessary to add `Style` name in the `const`, I think it's refactored styles

**Review by paulotijero — APPROVED** [review]

**Review by CesarCachay — APPROVED** [review]

**Pull request merged**